packages/core/src/__types__/method-types/import-export-params.ts

```diff
@@ -0,0 +1,56 @@
+// These types are separated from `method-types.ts` to avoid export conflicts.
+//
+// On mobile there's no XSS risk, so react-native-wallet-kit can perform import/export
```

Comment:
> // On mobile there's no XSS risk, so react-native-wallet-kit can perform import/export

not quite sure we can put 100% of our weight behind this claim :) 

https://medium.com/@srilakivarma/how-i-found-a-critical-stored-xss-in-a-financial-app-ios-android-eac1447e9702